frontend/src/components/Teams/CreateTeam/CardMember/index.tsx

```diff
@@ -35,6 +37,10 @@ const CardMember = React.memo<CardBodyProps>(
 		const setMembersList = useSetRecoilState(membersListState);
 		const membersList = useRecoilValue(membersListState);
```

Comment:
```suggestion
		const [membersList, setMembersList] = useRecoilState(membersListState)
```

You can use this method if you want to access and set the recoil value